locales/de/translation.json

```diff
@@ -103,5 +108,11 @@
   "faq_6_answer": "HTTPS kann mittels der Seite 'Settings' aktiviert werden. Aktuell verwendet Syncloud",
   "faq_6_answer_1": "und selbst-signierte Zertifikate. Alle Interaktionen mit Let's Encrypt sind automatisiert, sodass Sie sich um Nichts selbst kümmern müssen um in den Genuss einer verschlüsselten Verbindung zu kommen.",
   "faq_7": "Wo speichert Syncloud meine Daten?",
-  "faq_7_answer": "Syncloud speichert alle Daten entweder direkt auf Ihrem Syncloud-Gerät oder auf einer angeschlossenen externen Festplatte. Dies beinhaltet auch alle Daten der Syncloud-Anwendungen. Da Syncloud auch freie Gigabyte auf einer SD-Karte oder einem Flash-Speicher zum Ablegen der Daten verwenden kann, ist es für die Verwendung von Syncloud nicht zwingend notwendig eine externe Festplatte zu verwenden."
+  "faq_7_answer": "Syncloud speichert alle Daten entweder direkt auf Ihrem Syncloud-Gerät oder auf einer angeschlossenen externen Festplatte. Dies beinhaltet auch alle Daten der Syncloud-Anwendungen. Da Syncloud auch freie Gigabyte auf einer SD-Karte oder einem Flash-Speicher zum Ablegen der Daten verwenden kann, ist es für die Verwendung von Syncloud nicht zwingend notwendig eine externe Festplatte zu verwenden.",
+  "faq_8": "Kann ich mit SSH auf mein Gerät zugreifen?",
+  "faq_8_answer": "Ja. Die Standard-Anmeldedaten vor der Aktivierung sind Login: root, Passwort: syncloud. Nach der Aktivierung ist das Passwort identisch mit dem Gerätepasswort.",
+  "faq_9": "Kann ich mein Gerät auch ohne mobile Anwendung aktivieren?",
+  "faq_9_answer": "Ja. Sie müssen die Geräte-IP herausfinden (normalerweise auf Ihrer Router-Webseite), dann öffnen Sie http://&lt;IP&gt; in Ihrem Browser. Dann durchlaufen Sie die Aktivierungsprozedur.",
```

Comment:
1. i18next escapes special symbols, so no need to escape: http://&lt;IP&gt; => http://<IP>;
2. Also why do you need a semicolon at the end?

Reply:
1. Thank you, I didn't know that. I fixed this.
2. There were no semicolons, just the html entities.